src/components/ScaleBuilder.vue

```diff
@@ -450,7 +466,7 @@ async function doImport(importerKey: ImporterKey, event: Event) {
         "
       >
         <p><strong>Share scale as URL</strong></p>
-        <p>Open the sharing dialog</p>
+        <p>Copy this scale's unique URL for convenient sharing</p>
```

Comment:
I think URL sharing has higher priority and should be moved up in the list. Now I have to scroll to share my scale if I don't feel like copying from the address bar.